.github/workflows/releasebot.yml

```
@@ -0,0 +1,254 @@
+        with:
+          result-encoding: string
+          script: |
+            const { GITHUB_REF = 'main', GITHUB_REPOSITORY } = process.env;
+            const baseUrl = new URL(`https://github.com/${GITHUB_REPOSITORY}/blob/${GITHUB_REF}/`);
+            return baseUrl.toString();
+
+      - name: Get pnpm packages
+        id: pnpm-packages
+        run: |
+          echo "packages-output=$(pnpm list --recursive --depth -1 --json | jq -c 'map(select(.private == false))')" >> $GITHUB_OUTPUT
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)